hack/arktos-cni.rc

```diff
@@ -158,6 +164,25 @@ install_arktos_containerd() {
     fi
 }
 
+install_mizar_arktos_containerd () {
+    [ ! -d "$HOME/arktos-containerd" ] && mkdir $HOME/arktos-containerd
+    local CONTAINERD_DIR=$HOME/arktos-containerd
+    [ ! -f "$CONTAINERD_DIR/containerd" ] && wget -qO- https://github.com/CentaurusInfra/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > $CONTAINERD_DIR/containerd && sudo chmod +x $CONTAINERD_DIR/containerd
+    echo "Checking arktos containerd..."
+    if [ ! "$(${CONTAINERD_DIR}/containerd --version)" == "$(containerd --version)" ] ; then
+      echo "arktos containerd not found..."
+      [ ! -f "$HOME/arktos-containerd/containerd-1.4.2-linux-amd64.tar.gz" ] && wget https://github.com/containerd/containerd/releases/download/v1.4.2/containerd-1.4.2-linux-amd64.tar.gz -P $HOME/arktos-containerd/
+      sudo tar -zxf $HOME/arktos-containerd/containerd-1.4.2-linux-amd64.tar.gz --directory /usr/
+      echo "installing arktos containerd..."
+      sudo systemctl stop containerd
+      sudo mv /usr/bin/containerd /usr/bin/containerd.bak
+      sudo cp $HOME/arktos-containerd/containerd /usr/bin/
+      sudo systemctl start containerd
+      sudo systemctl restart docker
+    else
+      echo "arktos containerd found..."
+    fi
+}
```

Comment:
in kube-up env, we won't need the mizar specific containerd anymore, so this code is not needed anymore.

Reply:
In local env, containerd is used. Please see line 250 below in same file.